FlowCrypt/Core/CoreHost.swift

```diff
@@ -54,14 +55,15 @@ final class CoreHost: NSObject, CoreHostExports {
         return Cryptor(operation: .decrypt, algorithm: .aes, mode: .CFB, padding: .NoPadding, key: key, iv: iv).update(byteArray: ct)!.final()!
     }
 
-    // rsa verify is used by OpenPGP.js during decryption as well to figure out our own key preferences
-    // this slows down decryption the first time a private key is used in a session because bn.js is slow
-    // Using GMP C library modular exponentiation reduces rsa4096 verify time from 800ms to 40ms
```

Comment:
Please measure how long does it take to decrypt a message (first time, and second time) when this new library is in place, and compare that to speed of decrypting a message when you return an empty string here (in which case bn.js will have to compute it, likely slower).

Measure each thing twice because there is some caching going on.

Then please update this comment.

Reply:
Decrypting measurements: 
**First message**
1st run - 783 ms
2nd run - 251 ms

_with empty string_ 
1st run - 314 ms
2nd run - 138 ms

**Second message**
1st run - 510 ms
2nd run - 73 ms

_with empty string_ 
1st run - 318 ms
2nd run - 297 ms